app/services/vacancy_search_builder.rb

```diff
@@ -167,7 +167,7 @@ def keyword_multi_match(keyword)
     {
       multi_match: {
         query: keyword,
-        fields: %w[job_title^5 subject.name^3],
+        fields: %w[job_title^5 subject.name^3 first_supporting_subject.name^3 second_supporting_subject.name^3],
```

Comment:
I'd question whether the main subject should be weighted higher than the supporting subjects as by definition they are less important to the hiring staff, and therefore to the role itself. 

```suggestion
        fields: %w[job_title^5 subject.name^3 first_supporting_subject.name^2 second_supporting_subject.name^2],
```